src/pgduckdb_planner.cpp

```diff
@@ -87,14 +87,23 @@ CreatePlan(Query *query, bool throw_error) {
 
 		typtup = (Form_pg_type)GETSTRUCT(tp);
 
-		Var *var = makeVar(INDEX_VAR, i + 1, postgresColumnOid, typtup->typtypmod, typtup->typcollation, 0);
+		/* We fill in the varno later, once we know the index of the custom RTE
+		 * that we create. We'll know this at the end of DuckdbPlanNode. This
+		 * can probably be simplified when we don't call the standard_planner
+		 * anymore innside DuckdbPlanNodei, because then we only need a single
```

Comment:
vim user detected :-)
```suggestion
		 * anymore innside DuckdbPlanNode, because then we only need a single
```